nxt/backend/bluetooth.py

```diff
@@ -122,6 +122,8 @@ def find(self, host=None, name=None, **kwargs):
             else:
                 devhost, devname = dev, None
             if (host is None or devhost == host) and (name is None or devname == name):
+                if type(devhost) is bytes:
+                    devhost = devhost.decode('utf8')
```

Comment:
I think it should be done earlier, so that it can be compared with `host` in the `if` condition.